### ISSUE TITLE
Install debug symbols (PDB) on Windows (MSVC)

### ISSUE THREAD

**Issue by Youw** [libjpeg-turbo/libjpeg-turbo#324]
I think usefulness of this improvement is self-explaining.

CMake changes are minimal. E.g.:
```cmake
if(MSVC)
  install(FILES $<TARGET_PDB_FILE:turbojpeg>
    DESTINATION ${CMAKE_INSTALL_BINDIR} OPTIONAL)
endif()
```

I can prepare PR, if maintainers approve it.

**Comment by dcommander** [comment]
This is a good case in point for how fixing or implementing something in your specific development environment is only the first step toward integrating the fix/feature into a cross-platform product such as libjpeg-turbo.  In order to accept such a feature, I would need answers to the following questions:
1. Does the use of the `TARGET_PDB_FILE` generator expression raise the minimum CMake version requirements?  (Currently our build system supports CMake 2.8.12 and later.)
2. Does the new code properly deal with Release and MinSizeRel builds, under which no PDB files are generated?  I assume it does, due to the use of `OPTIONAL`, but this needs to be tested.  If it breaks for any reason with release builds, then the new code needs to be applied only when `CMAKE_BUILD_TYPE` is `Debug` or `RelWithDebInfo`.
3. Does the installation still work properly both from the command line (NMake) and from the Visual Studio IDE? (Multiple versions should be tested, including 2010.)
4. Does building the Windows installer still work properly?
5. Does the `uninstall` target still work properly?

**Comment by Youw** [comment]
Hm, good point(s).

1. As far as I can see in [this](https://gitlab.kitware.com/cmake/cmake/blob/80b9f0cbe263c64613acd18e4e94505924fce40e/Source/cmGeneratorExpression.cxx) commit - support for `TARGET_PDB_FILE` has been added with CMake 3.1.0.
I guess CMake version check can be added, if such solution acceptable.
2. Yes. `OPTIONAL` does its job.
3. Personally I tested NMake, Jom NMake, MSVC 2015 and MSVC 2017 generators - there was no issues. Currently I don't have other versions installed to check.
4. Never tried, but I see no reasons why not. `install(FILES ...)` is the simplest possible form of `install` command.
5. Yes.

Also, libjpeg-turbo uses `install(FILES ...)` to install headers. So everything that works for headers - will work for PDBs.
